WordPress/Classes/ViewRelated/Post/PostEditorNavigationBarManager.swift

```diff
@@ -0,0 +1,179 @@
+import Gridicons
+
+protocol PostEditorNavigationBarManagerDelegate: class {
+    var publishButtonText: String { get }
+    var isPublishButtonEnabled: Bool { get }
+    var uploadingButtonSize: CGSize { get }
+
+    func navigationBarManager(_ manager: PostEditorNavigationBarManager, closeWasPressed sender: UIButton)
+    func navigationBarManager(_ manager: PostEditorNavigationBarManager, moreWasPressed sender: UIButton)
+    func navigationBarManager(_ manager: PostEditorNavigationBarManager, blogPickerWasPressed sender: UIButton)
+    func navigationBarManager(_ manager: PostEditorNavigationBarManager, publishButtonWasPressed sender: UIButton)
+    func navigationBarManager(_ manager: PostEditorNavigationBarManager, displayCancelMediaUploads sender: UIButton)
+}
+
+// A class to share the navigation bar UI of the Post Editor.
+// Currenly shared between Aztec and Gutenberg
+//
+class PostEditorNavigationBarManager {
+    weak var delegate: PostEditorNavigationBarManagerDelegate?
+
+    // MARK: - Buttons
+
+    /// Dismiss Button
+    ///
+    lazy var closeButton: WPButtonForNavigationBar = {
+        let cancelButton = WPStyleGuide.buttonForBar(with: AztecPostViewController.Assets.closeButtonModalImage, target: self, selector: #selector(closeWasPressed))
+        cancelButton.leftSpacing = AztecPostViewController.Constants.cancelButtonPadding.left
+        cancelButton.rightSpacing = AztecPostViewController.Constants.cancelButtonPadding.right
```

Comment:
Considering the changes in this PR I'd say `cancelButtonPadding.left` and `cancelButtonPadding.right` should be moved to this class.